clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -2820,6 +2818,8 @@ class SyclKernelBodyCreator : public SyclKernelFieldHandler {
   // top-level pointers.
   uint64_t StructDepth = 0;
   VarDecl *KernelHandlerClone = nullptr;
+  bool IsSIMD = false;
```

Comment:
Let's keep it consistent.
```suggestion
  bool IsESIMD = false;
```

Reply:
I don't see where it is resolved.

Reply:
I did change it but looks like it was the other Visitor that I touched. I will fix this too.